android/src/SensorListener.java

```diff
@@ -61,6 +61,9 @@ void onGliderLinkTraffic(long gid, String callsign,
                            double gspeed, double vspeed,
                            double bearing);
 
+  void onFlarmTraffic(int AlarmLevel,int RelativeNorth,int RelativeEast,int RelativeVertical,
+                      String ID,int Track,double TurnRate,int GroundSpeed,double ClimbRate,
+                      int AcftType,boolean Stealth);
```

Comment:
Parameter names shouldn't use CamelCase
Why is `Track` and `GroundSpeed` an integer?